projects/distributed-tracing/src/shared/components/span-detail/span-data.ts

```diff
@@ -12,4 +13,6 @@ export interface SpanData {
   tags: Dictionary<unknown>;
   requestUrl: string;
   exitCallsBreakup?: Dictionary<string>;
+  startTime?: number;
```

Comment:
Surprised `startTime` isn't required. Is there a reason some of these properties optional?

Reply:
For `startTime` I've fixed. `logEvents` can be `undefined`. Apart from these I do not have much idea. may be same reason for others as well.